### PR TITLE
chore: update lab intros

### PR DESCRIPTION
**Pull request by zairahira**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

This PR updates the lab intros in `intro.json`.

Closes #XXXXX

### CODE REVIEW

File: client/i18n/locales/english/intro.json [hunk 1]
```diff
@@ -3360,7 +3409,8 @@
       "lab-mood-board": {
         "title": "Build a Mood Board",
         "intro": [
-          "In this lab, you will create a mood board using a React with props."
+          "In this lab, you will create a mood board using React props.",
```
Comment (ojeytonwilliams) [inline client/i18n/locales/english/intro.json:3412]:
```suggestion
          "In this lab, you will create a mood board using React.",
```
I think it's enough to mention props in the next step.

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]
I saw a couple of minor things. 

Also, I think we should be consistent about contractions and commas in lists. For example, we use both "You'll" and "You will". Also, I _believe_ it's standard in US English to write lists like this "stuff, whatsits, and more", rather than "stuff, whatsits and more".

**Comment by naomi-lgbt** [comment]
> Also, I believe it's standard in US English to write lists like this "stuff, whatsits, and more", rather than "stuff, whatsits and more".

This is called the Oxford Comma, and it's been a point of contention for many of our linguists for quite some time.

**Comment by zairahira** [comment]
Thanks for sharing the feedback Oliver.
For consistency, I'll make the following changes in the next commit:

- Change You will -> You'll
- Use the Oxford comma("stuff, whatsits, and more").

**Comment by ojeytonwilliams** [comment]
> > Also, I believe it's standard in US English to write lists like this "stuff, whatsits, and more", rather than "stuff, whatsits and more".
> 
> This is called the Oxford Comma, and it's been a point of contention for many of our linguists for quite some time.

Yeah, I was vaguely aware that there was some light controversy. A bit of googling suggests it's common in US English, though.

**Review by ojeytonwilliams — APPROVED** [review]
LGTM :+1: Thanks, Zaira!

**Review by moT01 — APPROVED** [review]
Lets go with it. 

I usually prefer the oxford comma - depends on the context though.
I also usually prefer not to use contractions - I think it sounds more professional.
But I'm fine with this for now 👍

**Comment by ojeytonwilliams** [comment]
@moT01 sure, that makes sense.  Can we update the contribution site with a style guide? If the curriculum team broadly agrees, that is.

**Pull request merged**